src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Interop/InternalRegisteredFunction.ts

```diff
@@ -1,5 +1,8 @@
-﻿import { invokeWithJsonMarshalling } from './InvokeWithJsonMarshalling';
```

Comment:
It looks like most of the `import` changes aren't needed. All the additions except `invokeWithJsonMarshallingAsync` and `invokePromiseCallback` can be reverted.